spec.html

```diff
@@ -35771,6 +36315,54 @@ <h1>
           </emu-alg>
         </emu-clause>
 
+        <emu-clause id="sec-runtime-semantics-maybesimplecasefolding-a" type="abstract operation">
+          <h1>
+            MaybeSimpleCaseFolding (
+              _A_: a CharSet,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It uses the Unicode <a href="https://www.unicode.org/reports/tr44/#Simple_Case_Folding">Simple Case Folding</a> mapping, which maps one code point to one code point, and which is abbreviated <a href="https://www.unicode.org/Public/UCD/latest/ucd/PropertyAliases.txt">scf</a>.</dd>
+          </dl>
+          <emu-alg>
+            1. If _UnicodeSets_ is *false* or _IgnoreCase_ is *false*, return _A_.
+            1. Let _S_ be a copy of _A_.
+            1. For each element _cs_ (a character or sequence of characters) in _A_, do
+              1. Create an empty sequence of characters _t_.
```

Comment:
```suggestion
              1. Let _t_ be an empty sequence of characters.
```

Reply:
This must have been done by an intervening commit.